src/main/java/me/botsko/prism/commands/WandCommand.java

```diff
@@ -17,262 +22,249 @@
-						.playerHeaderMsg("Restore wand " + ChatColor.RED + "disabled" + ChatColor.WHITE + "."));
-			}
-			else {
-				wand = new RestoreWand(plugin);
-				call.getPlayer().sendMessage(Prism.messenger.playerHeaderMsg(
-						"Restore wand " + ChatColor.GREEN + "enabled" + ChatColor.WHITE + wandOn + parameters + "."));
-				enabled = true;
-			}
-		}
-
-		/*
-		  Off
-		 */
-		else if (type.equalsIgnoreCase("off")) {
-			call.getPlayer().sendMessage(Prism.messenger
-					.playerHeaderMsg("Current wand " + ChatColor.RED + "disabled" + ChatColor.WHITE + "."));
-		}
-
-		// Not a valid wand
-		else {
-			call.getPlayer().sendMessage(Prism.messenger.playerError("Invalid wand type. Use /prism ? for help."));
-			return;
-		}
-
-		final PlayerInventory inv = call.getPlayer().getInventory();
-		if (enabled) {
-
-			if (item_material == null) {
-				if (Objects.equals(mode, "block")) {
-					item_material = Material.SPRUCE_LOG;
-				} else if (Objects.equals(mode, "item")) {
-					item_material = Material.STICK;
-				}
-				else {
-					item_material = Material.AIR;
-				}
-			}
-
-			wand.setWandMode(mode);
-			wand.setItem(item_material);
-
-			Prism.debug("Wand activated for player - mode: " + mode + " Item:" + item_material);
-
-			// Move any existing item to the hand, otherwise give it to them
-			if (plugin.getConfig().getBoolean("prism.wands.auto-equip")) {
-				if (!InventoryUtils.moveItemToHand(inv, item_material)) {
-					// Store the item they're holding, if any
-					wand.setOriginallyHeldItem(inv.getItemInMainHand());
-					// They don't have the item, so we need to give them an item
-					if (InventoryUtils.handItemToPlayer(inv, new ItemStack(item_material, 1))) {
-						wand.setItemWasGiven(true);
-					}
-					else {
-						call.getPlayer().sendMessage(
-								Prism.messenger.playerError("Can't fit the wand item into your inventory."));
-					}
-				}
-				InventoryUtils.updateInventory(call.getPlayer());
-			}
-
-			// Let's build the QueryParameters for it if it's a Query wand.
-			if (wand instanceof QueryWandBase) {
-				if (!((QueryWandBase) wand).setParameters(call.getPlayer(), call.getArgs(), (isInspect ? 1 : 2))) { // This
-																													// returns
-																													// if
-																													// it
-																													// was
-																													// successful
-					call.getPlayer().sendMessage(Prism.messenger.playerError("Notice: Only some parameters used.."));
-				}
-			}
-
-			// Store
-			Prism.playersWithActiveTools.put(call.getPlayer().getName(), wand);
-		}
-		else {
-			if (oldwand != null) {
-				oldwand.disable(call.getPlayer());
-			}
-		}
-	}
-
-	@Override
-	public List<String> handleComplete(CallInfo call) {
-		return null;
-	}
+    private final Prism plugin;
+
+    /**
+     * Constructor.
+     * @param plugin Prism
+     */
+    public WandCommand(Prism plugin) {
+        this.plugin = plugin;
+    }
+
+     // TODO break this down.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Comment has incorrect indentation level 5, expected is 4, indentation should be the same level as line 36.](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5438808)